config/config.go

```diff
@@ -548,6 +550,7 @@ func DefaultMempoolConfig() *MempoolConfig {
 		// ABCI Recheck
 		Size:      5000,
```

Comment:
Probably we need to be clear now with units, i.e., when we talk about number of txs and when we talk about size in bytes.